tests/sklearn/test_sklearn_models.py

```diff
@@ -2032,3 +2033,63 @@ def test_error_raise_unsupported_pandas_values(model_class, bad_value, expected_
 
     with pytest.raises(ValueError, match=expected_error):
         model.fit(x_train, y_train)
+
+
+@pytest.mark.parametrize(
+    "model_class, parameters",
+    get_sklearn_linear_models_and_datasets()
+    + get_sklearn_tree_models_and_datasets()
+    + get_sklearn_neighbors_models_and_datasets(),
+)
+def test_initialization_variables_match(model_class, parameters, load_data, is_weekly_option):
+    """Test that CML models can be initialized with the same parameters scikit-learn models."""
+    n_bits = N_BITS_THRESHOLD_FOR_SKLEARN_CORRECTNESS_TESTS
```

Comment:
we usually use  `min(N_BITS_REGULAR_BUILDS)` although we actually have `get_n_bits_non_correctness(model_class)` , we don't really care about n_bits here anyway

Reply:
Ok I can use the min(N_BITS_REGULAR_BUILDS).

Reply:
I think we should better use `get_n_bits_non_correctness(model_class)` as it does `min(N_BITS_REGULAR_BUILDS)` but handles KNN differently, but I won't insist too much here because it seems we usually do `min(N_BITS_REGULAR_BUILDS)` anyway

Reply:
You are right we should use the utils function and adapt them to our needs. We should probably make sure we use them everywhere.

Reply:
Done